### DISCUSSION TITLE
Monte Carlo Thermal simulation for the Coalsack Region

### DISCUSSION THREAD

**Discussion by cyber-prags** [dullemond/radmc3d-2.0#36] [Q&A]
I am trying to model the three brightest stars in the Coalsack Region namely alpha cru, beta cru and beta cen and calculate do a thermal Monte Carlo simulation to see how the temperature changes with distance and observe the IR intensity at 100 microns. But my program throws an error everytime. I am attaching below the parameters that I used and my sample program. Please have a look at it and help me out. Its kind of urgent. 

[Sample program.txt](https://github.com/dullemond/radmc3d-2.0/files/11611314/Sample.program.txt)

	
[Coalsack parameters.txt](https://github.com/dullemond/radmc3d-2.0/files/11611306/Coalsack.parameters.txt)

**Comment by dullemond** [comment]
Dear cyber-prags,
A number of the input files you write out have the wrong format... You may want to re-start from the original run_simple_1/ model, and make changes one tiny step at a time, each time checking that things are still running. You can also look into the code itself to see what file format it expects.
Best wishes,
Kees
